packages/playwright-core/types/structs.d.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import { JSHandle, ElementHandle, Frame, Page, BrowserContext, Locator } from './types';
+import { JSHandle, ElementHandle, Frame, Page, BrowserContext } from 'playwright-core';
```

Comment:
importing playwright-core from playwright-core feels wrong. Why was it changed from ./types?

Reply:
This makes it possible to `declare module 'playwright-core'` and make this file pick up the new definitions. Otherwise, the regular and overwritten types `Page` do not match.